### PR TITLE
chore: P256 verify cost action test

### PR DESCRIPTION
**Pull request by meyer9**
This PR verifies that P256 verify precompile gas cost changed after the fork.

### CODE REVIEW

File: actions/harness/tests/base_v1_activation.rs
Comment (refcell) [inline actions/harness/tests/base_v1_activation.rs:1]: This file is starting to get a little long, can we possible break up?

Comment (meyer9) [reply actions/harness/tests/base_v1_activation.rs:1]: Yeah we can. Generally, I don't really mind large files if it semantically makes sense to group them. But, I understand it can make sense to split them up into individual files. I think we can also improve the interfaces available to action tests to make this shorter/reduce duplicate code. I might try doing that once these tests are merged.

File: actions/harness/tests/base_v1/p256.rs [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        gas_delta_post = acct.storage.get(&P256_GAS_DELTA_SLOT).copied().unwrap_or(U256::ZERO);
+        assert_eq!(success, U256::from(1), "P256VERIFY must succeed post-fork");
+    }
+
+    // The base gas fee doubles from 3,450 to 6,900 at Base V1.
+    assert!(
+        gas_delta_post > gas_delta_pre,
+        "post-fork P256VERIFY gas delta ({gas_delta_post}) must exceed pre-fork delta \
+         ({gas_delta_pre}) due to doubled base gas fee"
+    );
```
Comment (danyalprout) [inline actions/harness/tests/base_v1/p256.rs:133]: nit: maybe nice to expect specific amounts here

**Review by danyalprout — APPROVED** [review]

File: actions/harness/tests/base_v1/p256.rs (hunk 1 above)
Comment (meyer9) [reply actions/harness/tests/base_v1/p256.rs:133]: will do in a follow up PR with expmod also

**Comment by cb-heimdall** [comment]
<h1>✅ Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
✅
<code>1/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td><code>2</code> if repo is sensitive</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>1</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>2</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Review by danyalprout — DISMISSED** [review]

**Review by danyalprout — APPROVED** [review]
